pages/linux/tshark.md

```diff
@@ -6,6 +6,10 @@
 
 `tshark`
 
+- Capture only specific packets:
```

Comment:
By taking a look at the example I assume that with "specific" you mean packages going through port 53. 

IMHO it makes sense to have this clearly stated. 
The word "Specific" here does not explain much about what is the filter.

Reply:
Perhaps something like:

> Capture only packet matching a specific wireshark filter:

would read better?

Reply:
That is nearly the same as the description after. I would prefer to clearly distinguish between -f and -Y.

Reply:
@agnivade welp, that's a good point, I agree. I got confused the first time I read this PR because I thought the two examples were doing the exact same thing. The result is the same, but what the program does is very different.

Reply:
I've added more explicit mention of `capture filter` and `output filter`. Hopefully that wlll clear the first time confusion.